start_kafka.sh

```diff
@@ -1,10 +1,25 @@
 #!/bin/bash
 
+if [ $# -eq 0 ]; then
+    echo "USAGE: start_kafka.sh start|stop|kill"
+    exit 1
+fi
+
 if [ "x$KAFKA_HOME" = "x" ]; then
   echo "Please set KAFKA_HOME env variable to the kafka install directory"
   exit 1
 fi
 
+if [ ! -x "$(command -v nc)" ]; then
+  echo "Please install Netcat (nc) first"
+  exit 1
+fi
+
+if [ ! -x "$(command -v java)" ]; then
```

Comment:
In case we're getting into checking whether java is installed, it would also be nice to check whether it's 8+

Reply:
That was added.